olm-catalog/serverless-operator/index/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM openshift/origin-base as builder
 
-COPY --from=quay.io/operator-framework/opm:latest /bin/opm /bin/opm
+COPY --from=quay.io/operator-framework/opm:v1.23.2-amd64 /bin/opm /bin/opm
```

Comment:
`:master` is multi arch, can we use that one?

Reply:
or `v1.23`

Reply:
Is it safe to use master? that tag is multi arch too.

Reply:
`v1.23.2-amd64` isn't `v1.23` is multi arch

Reply:
> Is it safe to use master?

No, it's not but so far we have used "latest" which isn't safe as well

Reply:
I'd go with `v1.23`